apps/masterbots.ai/types/types.ts

```
@@ -20,11 +21,12 @@ export interface Chat extends Record<string, any> {
 }
 
-export type AiToolCall = {
-  toolCallId: string
-  toolName: WordWareFlowPaths
-  args: Record<string, any>
-}
+export type AiToolCall = ToolCall &
+  FunctionToolCall & {
+    toolCallId: string
+    toolName: WordWareFlowPaths
+    args: Record<string, any>
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)